lib/AlgoQuant.js

```diff
@@ -1,24 +1,19 @@
-import axios from "axios";
+import { User } from "./user";
 
-export class AlgoQuant {
-  apiTest(token) {
-    const API_URL =
-      "https://55vb89e0gb.execute-api.us-east-1.amazonaws.com/Development";
+// Create new User object
+const user = new User();
```

Comment:
This should be done in the constructor of the clas.

Reply:
done